app/controllers/projects_controller.rb

```diff
@@ -15,6 +15,20 @@ def index
   # GET /projects/1
   # GET /projects/1.json
   def show
+    # all exisiting project are null, we ensure private projects are explicitly set to 0
+    public_access = @project.public_access.nil? ? 1 : @project.public_access
```

Comment:
I think it's a better default to set the variable to 0 when it is nil

Reply:
I assume 0 means private right? so that means all projects are private by default?

Reply:
Yes. Until we set them to public explicitly

Reply:
Okay, updating ...